utils/pruning.py

```diff
@@ -167,8 +167,10 @@ def prune_metadata_str(question, db_name, public_data=True):
     root_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
     emb_path = os.path.join(root_dir, "data", "embeddings.pkl")
     if public_data:
+        print("Loading public data")
         import defog_data.supplementary as sup
     else:
+        print("Loading private data")
```

Comment:
do not import private data unconditionally or make this a standard requirement, since by default most people using this would be relying on the OSS public data.